modules/virt-using-disk-image-install-windows.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-automating-windows-sysprep.adoc
+
```

Comment:
Please add the Jupiter attributes to all the files that don't have them yet (see [this doc](https://docs.google.com/document/d/1o4rrsgowUzGiVSH84Wg-Tt4DxmDXE_hg2H946NnXj6s/edit))